src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const th = table.tHead.rows[0].cells[1];
+const targetTh = table.tHead.rows[0].cells[4];
+const newTh = table.tHead.rows[0].insertCell(5);
+
+newTh.outerHTML = targetTh.outerHTML;
+targetTh.innerHTML = th.innerHTML;
+
+const tf = table.tFoot.rows[0].cells[1];
+const targetTf = table.tFoot.rows[0].cells[4];
+const newTf = table.tFoot.rows[0].insertCell(5);
```

Comment:
Similar to the header, inserting a new cell at index 5 in the footer assumes a certain table structure. Verify the number of columns to prevent potential issues.